lib/matplotlib/axes/_axes.py

```diff
@@ -3889,17 +3889,25 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
         ----------------
         kwargs : `~matplotlib.collections.Collection` properties
 
+        See Also
+        --------
+        plot : to plot scatter plots when markers are identical in size and
+            color
+
         Notes
-        ------
-        Any or all of `x`, `y`, `s`, and `c` may be masked arrays, in
-        which case all masks will be combined and only unmasked points
-        will be plotted.
+        -----
+
+        * When attempting to plot a scatter plots where markers don't vary in
```

Comment:
I think this construction can just be "the `plot` function will be faster for scatterplots where markers don't vary in size or color.